src/utils/index.js

```diff
@@ -54,10 +54,14 @@ export function setVersion(version) {
 }
 
 export function getVersion() {
-  if (/version=(\d+\.?\x?)+/.test(location.href)) {
-    return location.href.match(/version=(\d+\.?\x?)+/)[0];
+  try {
+    if (/version=(\d+\.?\x?)+/.test(location.href)) {
+      return location.href.match(/version=((\d+\.?\x?)+)/)[1];
+    }
+  } catch (error) {
+    console.error(error)
+    return "master";
```

Comment:
@urfreespace It seems now if we receive `version=master`, we return `undefined` instead of `master`?

This seems to be the cause of https://github.com/apache/pulsar/issues/19681.

cc @dave2wave